document-storage-adapter/src/Pssg.DocumentStorageAdapter/Controllers/FileController.cs

```diff
@@ -110,7 +110,7 @@ public async Task<ActionResult> Download([FromBody] ViewModels.Download download
                 {
                     // Check the file type is .tiff by filename
 
-                    if (fileName.Contains(".tiff"))
+                    if (fileName.ToLower().Contains(".tiff") || fileName.ToLower().Contains(".tif"))
```

Comment:
please remove the redundant check for .tiff, as the check for .tif is sufficient.